kubectl-minio/cmd/helpers/constants.go

```diff
@@ -42,7 +42,7 @@ const (
 	DefaultTenantImage = "minio/minio:RELEASE.2023-01-12T02-06-16Z"
 
 	// DefaultKESImage is the default KES image used while creating tenant
-	DefaultKESImage = "minio/kes:v0.18.0"
+	DefaultKESImage = "minio/kes:v0.22.3"
```

Comment:
Latest KES version https://github.com/minio/kes/releases/tag/2023-02-15T14-54-37Z

```suggestion
	DefaultKESImage = "minio/kes:2023-02-15T14-54-37Z"
```